src/wechaty.ts

```diff
@@ -610,15 +610,22 @@ export class Wechaty extends EventEmitter implements Sayable {
   private addListenerFunction (event: WechatyEventName, listener: AnyFunction): void {
     log.verbose('Wechaty', 'addListenerFunction(%s)', event)
 
+    const handleError = (e: Error, type = '') => {
+      log.error('Wechaty', 'addListenerFunction(%s) listener %sexception: %s', event, type ? '' : (type + ''), e)
```

Comment:
Seems like the `type` is a string and will always be a string, so I don't think we need to do `type ? '' : (type + '')` here, which is redundant.

```suggestion
      log.error('Wechaty', 'addListenerFunction(%s) listener %s exception: %s', event, type, e)
```

Reply:
Have no idea why I did it like this. lol